Project.toml

```diff
@@ -0,0 +1,27 @@
+name = "Mux"
```

Comment:
Should remove the `REQUIRE` file in the root directory after making these changes.

Reply:
Good catch!